src/openforms/submissions/status.py

```diff
@@ -0,0 +1,65 @@
+"""
+Utility to interact with the celery task status.
+"""
```

Comment:
Very much WIP still - it works, sort of, but the `payment_url` for example is not relevant and probably while implementing it in the SDK, we will discover new nuances.

Reply:
See comment about status/result